rpm/SOURCES/etc/logrotate.d/logrotate-contextAdapter.conf

```diff
@@ -0,0 +1,29 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of the iotagent-thinking-things.
```

Comment:
I saw some references to iotagent-context-adapterin this file... I guess you mean context-adapter right? :)